src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +29,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        logger.info("successful fetched data from DB");
```

Comment:
The log message 'successful fetched data from DB' is not accurate since the data is hardcoded and not fetched from a database. It should either be removed or corrected to reflect the actual action being performed.